src/world/script_api/push_blocks.c

```diff
@@ -188,15 +188,14 @@ API_CALLABLE(func_80282E30) {
     script->varTable[4] = yThing = entity->position.y;
     script->varTable[5] = entityZ = entity->position.z;
 
-    xThing = (xThing - grid->centerPos.x);
+    xThing = xThing - grid->centerPos.x;
+    entityZ = entityZ - grid->centerPos.z;
```

Comment:
The previous steps here seem cleaner to me. Was this necessary to keep the match?

Reply:
it's not a match yet - just some WIP progress on the function